src/shared/lib/utils/dateTime.ts

```diff
@@ -200,3 +200,10 @@ export const getMonthAgoDate = () => formatToDtoDate(subMonths(new Date(), 1));
 
 export const buildDateTimeFromDto = (yyyymmdd: string, hhmmss: string) =>
   new Date(`${yyyymmdd} ${hhmmss}`);
+
+export const getDateFromString = (dateString: string) => {
+  // converts yyyy-mm-dd string to Date , ignoring timeZone
+  const [year, month, day] = dateString.split('-');
```

Comment:
It's worth adding an implicit check if dateString corresponds to the format you specified here.
To do that, you could check if all these variables have values. If not, throw Error